crates/rolldown/examples/watch.rs

```diff
@@ -1,28 +1,17 @@
 use std::sync::Arc;
 
-use rolldown::{Bundler, BundlerOptions, ExperimentalOptions, InputItem, ResolveOptions, Watcher};
-use rolldown_workspace::root_dir;
+use rolldown::{Bundler, BundlerOptions, ExperimentalOptions, Watcher};
+use sugar_path::SugarPath;
 use tokio::sync::Mutex;
 
 // cargo run --example watch
 
 #[tokio::main]
 async fn main() {
   let bundler = Bundler::new(BundlerOptions {
-    input: Some(vec![InputItem {
-      name: Some("rome-ts".to_string()),
-      import: root_dir().join("tmp/bench/rome/src/entry.ts").to_str().unwrap().to_string(),
-    }]),
-    cwd: Some(root_dir().join("tmp/bench/rome")),
+    input: Some(vec!["./entry.js".to_string().into()]),
+    cwd: Some(rolldown_workspace::crate_dir("rolldown").join("./examples/basic").normalize()),
```

Comment:
This change is intentional, use a small test case to make the debugging easier in watch mode.